src/services/role/hooks/index.js

```diff
@@ -2,43 +2,19 @@
 
 const globalHooks = require('../../../hooks');
 const hooks = require('feathers-hooks');
-const auth = require('feathers-authentication').hooks;
+const auth = require('feathers-authentication');
 
 exports.before = function(app) {
 	return {
-		all: [],
-		find: [
-			auth.verifyToken(),
-			auth.populateUser(),
-			auth.restrictToAuthenticated()
-		],
-		get: [
-			/*auth.verifyToken(),
-			 auth.populateUser(),
-			 auth.restrictToAuthenticated()*/
-		],
+		all: [auth.hooks.authenticate('jwt')],
```

Comment:
We should check if authentication should be required for getting information about roles